### DISCUSSION TITLE
docs/migrate-from-starter-theme/

### DISCUSSION THREAD

**Discussion by giscus** [hbstack/theme-cards#584] [General]
# docs/migrate-from-starter-theme/

本指南介绍如何从 HB 初始主题迁移到 HB 卡主题。

https://www.guoyinpm.com/docs/migrate-from-starter-theme/

**Comment by razonyang** [comment]
@gyculture thanks for using this theme, you need to replace the `giscus*` config on `params.yaml`, to save the comments on your own repository, and GitHub will send a notification to you if someone comment.

**Reply by gyculture** [reply]
Sorry and proceed. Thanks for your great job.
